statsmodels/graphics/tsaplots.py

```diff
@@ -64,9 +70,10 @@ def _plot_corr(
         lags = lags.astype(float)
         lags[0] -= 0.5
         lags[-1] += 0.5
-        ax.fill_between(
```

Comment:
That was my favorite part of this PR, but okay. I will probably open a separate PR for this, with reference to this one.

Reply:
I also like the change to use the dashed lines rather than the fill_between. We'll be able to better evaluate that change in its own PR, so thanks for splitting it out!